openadapt/record.py

```diff
@@ -628,11 +628,44 @@ def record(
 
     # TODO: discard events until everything is ready
 
+    # Variables for sequence detection
+    sequence = ['o', 'a', '.', 's', 't', 'o', 'p']
```

Comment:
What do you think about moving this to `config.STOP_SEQUENCES` (a list of possible sequences)?